pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -300,6 +300,12 @@ func (c *InstallerController) manageInstallationPods(operatorSpec *operatorv1.St
 
 		// if we are in a transition, check to see whether our installer pod completed
 		if currNodeState.TargetRevision > currNodeState.CurrentRevision {
+			// Skip creating the installer pod when there is already a revision higher than target revision
+			// In this case, wait for the revision controller to update the target revision on node.
+			if operatorStatus.LatestAvailableRevision > currNodeState.TargetRevision {
```

Comment:
expecting a test to reflect this? I thought this has good test coverage.

Reply:
this function has zero test coverage... I added tests in last commit.